aviary/variable_info/functions.py

```diff
@@ -144,13 +145,17 @@ def name_filter(name):
             group.promotes(comp.name, inputs=in_var_names, outputs=comp_promoted_outputs)
 
     if overridden_outputs:
-        print("\nThe following variables have been overridden:")
+        if aviary_inputs.get_val(Settings.VERBOSITY).value >= 1:
+            print("\nThe following variables have been overridden:")
         for prom_name in sorted(overridden_outputs):
-            print(f"  '{prom_name}")
+            item = aviary_inputs.get_item(prom_name)
+            print(f"  '{prom_name}  {item[0]}  {item[1]}")
 
     if external_overridden_outputs:
-        print("\nThe following variables have been overridden by an external subsystem:")
+        if aviary_inputs.get_val(Settings.VERBOSITY).value >= 1:
+            print("\nThe following variables have been overridden by an external subsystem:")
         for prom_name in sorted(external_overridden_outputs):
+            # do not print values because they will be updated by an external subsystem later.
             print(f"  '{prom_name}")
```

Comment:
I think you will want to indent the "for" and "print" lines (157-159), otherwise the names will print without context when verbosity = 0

Reply:
same with the other overriden_outputs section

Reply:
Done.